### ISSUE TITLE
[Bug] Accessibility text in dark mode not visible

### ISSUE THREAD

**Issue by mehershiri** [agamjotsingh18/trendhora#232]
### What happened?

The accessibility page text is not clearly visible due to the usage of dark text color with a dark Background making it difficult for the users to read

### Steps to Reproduce

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/7fa16fa6-77ac-4603-b891-6e48643129ad" />

**Comment by agamjotsingh18** [comment]
Too easy issue
